scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1042,7 +1042,6 @@ def align_val(val, size):
     return (int(val/size) * size)
 
 def update_bseg_size(self):
-    dev_count = 0
```

Comment:
Isn't initialization required here for dev_count ?

Reply:
This variable is not being used in the function, hence no need to define it.

Reply:
ok